dimitriosv/week3/dist/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Deep dive into React</title>
+    <script src="http://fb.me/react-0.14.7.js"></script>
+    <script src="http://fb.me/react-dom-0.14.7.js"></script>
```

Comment:
you do not need these any-longer because you have bundled them in your application already.